019-ThisOldDataWarehouse/Coach/Solutions/Challenge1/readme.md

```diff
@@ -10,30 +10,42 @@ be showcasing how to migrate your traditional SQL Server (SMP) to Azure Synapse
+
 WWI runs their existing database platforms on-premise with SQL Server 2017.  There are two databases samples for WWI.  The first one is for their Line of Business application (OLTP) and the second
-is for their data warehouse (OLAP).  You will need to setup both environments as our starting point in the migration.  Recommended to have students start Challenge 0 with setup of SQL VM before starting any presentations. This spin-up time is approx 30 mins and this will give you sufficient time to kick-off the event while their VMs are being setup.
-
-1. If you do not have a on-premise SQL Server 2017, you can provision a Azure Virtual Machine running SQL Server 2017 using this [Step by step guidance](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sql/virtual-machines-windows-portal-sql-server-provision) Recommended size is DS12
-    * Turn off IE Enhanced Security Config in [Server Manager](https://medium.com/tensult/disable-internet-explorer-enhanced-security-configuration-in-windows-server-2019-a9cf5528be65)
-    * Go to Windows Firewall internal to the VM and open a inbound port to 1433. This is required for SSIS Runtime to access the database.
-    * Go to Network Security Group (Azure) and setup inbound ports with 1433
-2. Download both WWI databases (Enterprise Edition) to your on-premise SQL server or Azure VM you have just provisioned. [Download Link](https://github.com/Microsoft/sql-server-samples/releases/tag/wide-world-importers-v1.0). Go to the section called, "SQL Server 2016 SP1 (or later) Any Edition aside from LocalDB; SQL Server 2016 RTM (or later) Evaluation/Developer/Enterprise Edition" and download the two bullets under this heading.
->The file names are WideWorldImporters-Full.bak and WideWorldImportersDW-Full.bak.  
->These two files are the OLTP and OLAP databases respectively.
-> Copy these two files to this directory on the Virtual machine C:\Program Files\Microsoft SQL Server\MSSQL14.MSSQLSERVER\MSSQL\Backup
-3. Follow this [Install and Configuration Instrution for the OLTP database](https://docs.microsoft.com/en-us/sql/samples/wide-world-importers-oltp-install-configure?view=sql-server-ver15)
-4. Follow this [Install and Configuration Instrution for the OLAP database](https://docs.microsoft.com/en-us/sql/samples/wide-world-importers-dw-install-configure?view=sql-server-ver15)
+is for their data warehouse (OLAP).  You will need to setup both environments as our starting point in the migration.  Recommended to have students start Challenge 0 with setup of SQL environment before starting any presentations. 
+
+1. Open your browser and login to your Azure Tenant.  We plan to setup the Azure Services required for the What the Hack (WTH).  In your portal, open the [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+
+2. Go into the cloud shell and select the subscription you plan to use for this WTH.
+
+        az account set --subscription {"Subscription Name"}
```

Comment:
I would wrap all these code blocks in triple tildes: ```. Such as:

```
az account show
az group create
```

Reply:
Not sure if you can view source on it, but it's:

--insert triple tilde here--
az account show
az group create
--insert another triple tilde here--

Reply:
Updated code blocks with tilde